.pre-commit-config.yaml

```diff
@@ -15,7 +15,42 @@ repos:
         # the results into docker.
         entry: bash -c 'git diff master -U0 | docker-compose run --rm home ./scripts/flake8-diff.sh'
         language: system
+
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.1
+    hooks:
+    -   id: check-yaml
+    # -   id: end-of-file-fixer
+    # -   id: requirements-txt-fixer
+    # -   id: trailing-whitespace
```

Comment:
Why are these commented out?

Reply:
Because I want to remember to add them but do not want to touch too many files in a single pull request.